DataProcessing/data_tagger.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pickle
+from collections import defaultdict
+from matplotlib import pyplot as plt
+
+
+# todo all of these actions need to be done to the db, currently they all run in O(n) for each Crop changed !
+from DataProcessing.dataProcessingConstants import ID_TO_NAME
+
+
+def mark_vague(db, track, crop_inds):
```

Comment:
Let's rename all these references of `db` to `pkl` so we don't get confused once we insert the real DB.